src/pages/extending-chat-widget/chat-widget-adapters/vue-adapter/index.mdx

```diff
@@ -0,0 +1,178 @@
+---
+weight: 30
+category: "extending-chat-widget"
+subcategory: "chat-widget-adapters"
+title: "Vue adapter"
```

Comment:
Hm, I'm pretty sure we should capitalize Adapter in the title - we're still thinking about one of the Chat Widget Adapters, not about any adapter. Same for other docs